session/create_consumer.go

```diff
@@ -24,17 +24,17 @@ import (
 	"github.com/mysteriumnetwork/node/identity"
 )
 
-// Manager defines methods for session management
-type Manager interface {
-	Create(consumerID identity.Identity, proposalID int) (Session, error)
-}
-
 // createConsumer processes session create requests from communication channel.
 type createConsumer struct {
-	SessionManager Manager
+	SessionManager Creator
```

Comment:
SessionManager var of type Creator sounds weird. Also it's not needed to be public

Reply:
fixed